lib/application.js

```diff
@@ -124,6 +124,9 @@ app.model = function(Model, config) {
   if (arguments.length > 1) {
     config = config || {};
     configureModel(Model, config, this);
+    this.on('modelRemoted', function() {
+      setSharedMethodSharedProperties(Model, this, config);
```

Comment:
The other issue is that the model config only lives for the duration of `app.model` run. I stuck it on the `Model.config` property so I could get to it from `loopback.js` but this is probably not the best way to do it (and is causing another test to fail). Is there a better way to make the `config` available on the model itself at a later time?